File: students/students/src/MON.SPA/src/models/environmentCharacteristics/studentRelativeModel.js
import { DropDownModel } from '@/models/dropdownModel';

export class StudentRelativeModel {
    constructor(obj = {}) {
      this.id = obj.id || 0;
      this.relativeType = obj.relativeType || null;
      this.notes = obj.notes || '';
      this.workStatus = obj.workStatus || null;
      this.description = obj.description || '';
      this.firstName = obj.firstName || '';
      this.middleName = obj.middleName || '';
      this.lastName = obj.lastName || '';
      this.address = obj.address || '';
      this.pinType = obj.pinType || new DropDownModel(); //ЕГН
      this.pin = obj.pin || '';
      this.index = obj.index || '';
      this.email = obj.email || '';
      this.phoneNumber = obj.phoneNumber || '';
      this.educationType = obj.educationType || null;
    }
  }

File: students/students/src/MON.Models/StudentModels/StudentRelativeModel.cs
namespace MON.Models.StudentModels
{
    public class StudentRelativeModel
    {
        public int Id { get; set; }
        public int ChildId { get; set; }
        public int PersonId { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Description { get; set; }
        public DropdownViewModel RelativeType { get; set; }
        public string Notes { get; set; }
        public DropdownViewModel WorkStatus { get; set; }
        public DropdownViewModel PinType { get; set; }
        public string Pin { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public DropdownViewModel EducationType { get; set; }
    }
}
